### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by maksym-nemera**
[DEMO LINK](https://maksym-nemera.github.io/react_movies-list/)

### CODE REVIEW

File: src/components/MovieCard/MovieCard.tsx [hunk 1]
```diff
@@ -1,7 +1,51 @@
...
+import { Movie } from '../../types/Movie';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
+  movie: Movie;
+};
+
+export const MovieCard: React.FC<Props> = ({ movie }) => (
+  <>
```
Comment (ruslanpashkov) [inline src/components/MovieCard/MovieCard.tsx:11]: do we need `React.Fragment`, when we can take our wrapper with `className="card"`?

File: src/components/MovieCard/MovieCard.tsx [hunk 2]
```diff
@@ -1,7 +1,51 @@
...
+      <div className="media">
+        <div className="media-left">
+          <figure className="image is-48x48">
+            <img src="images/imdb-logo.jpeg" alt="imdb" />
+          </figure>
+        </div>
+
+        <div className="media-content">
+          <p className="title is-8" data-cy="MovieTitle">
+            {movie.title}
```
Comment (ruslanpashkov) [inline src/components/MovieCard/MovieCard.tsx:32]: use destructuring to take properties from `movie` object

**Review by ruslanpashkov — CHANGES_REQUESTED** [review]

File: src/components/MoviesList/MoviesList.tsx [hunk 3]
```diff
@@ -1,6 +1,18 @@
...
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <div className="card" data-cy="Movie" key={movie.imdbId}>
+        <MovieCard movie={movie} />
+      </div>
+    ))}
```
Comment (naziamoff) [inline src/components/MoviesList/MoviesList.tsx:16]: don't forget about semantics. Here you have a list with list items, not just random divs

**Review by naziamoff — APPROVED** [review]
looks good 👍

**Pull request closed without merging**